src/lib/services/api-token-service.ts

```diff
@@ -83,15 +80,16 @@ export class ApiTokenService {
     }
 
     public async creteApiToken(
```

Comment:
Why does Crete enter into it, annoying typo here.
```suggestion
    public async createApiToken(
```